_posts/en/newsletters/2020-08-05-newsletter.md

```diff
@@ -0,0 +1,151 @@
+---
+title: 'Bitcoin Optech Newsletter #109'
+permalink: /en/newsletters/2020/08/05/
+name: 2020-08-05-newsletter
+slug: 2020-08-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter contains our regular sections with recently
+transcribed talks and conversations, releases and release candidates,
+and notable changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+*No significant news about Bitcoin infrastructure development this
+week.*
+
+## Recently transcribed talks and conversations
+
+*[Bitcoin Transcripts][] is the home for transcripts of technical
+Bitcoin presentations and discussions. In this monthly feature, we
+highlight a selection of the transcripts from the previous month.*
+
+FIXME:michaelfolkson
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
+
+- [C-Lightning 0.9.0][C-Lightning 0.9.0] is the newest major version of
+  C-Lightning.  It adds support for the updated `pay` command and new
+  `keysend` RPC described in [Newsletter #107][news107 notable].  It
+  also includes several other notable changes and multiple bug fixes.
+
+- [Bitcoin Core 0.20.1][Bitcoin Core 0.20.1] is a new maintenance
+  release.  In addition to bug fixes and some RPC behavior changes
+  resulting from those fixes, the planned release provides compatibility
+  with recent versions of [HWI][topic HWI] and its support for hardware
+  wallet firmware released for the [fee overpayment attack][].
+
+- [LND 0.11.0-beta.rc1][LND 0.11.0-beta] is the release candidate for a
+  new major release.  <!-- FIXME:harding to update Tuesday (LND haven't
+  yet published their draft release notes) -->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19569][] allows Bitcoin Core to fetch the parents of _orphan_
```

Comment:
nit: can probably leave "orphan" unitalicized

Reply:
I think it's ok to leave this italicized if the term is not expected to be familiar to our readers. Let's see what other reviewers think.

Reply:
No opinion; either way is fine by me.